lagunita-gingko/lms/static/sass/partials/base/custom_variables/images.scss

```diff
@@ -0,0 +1 @@
+$homepage-bg-image: url('/static/lagunita/images/lake_lag.jpg');
```

Comment:
this is currently pulling the image from lagunita theme, so once this theme becomes lagunita theme, it should just work. Though if we can find a better way to set a image url in SCSS I'd prefer it.